infra/Pos/Communication/Util.hs

```diff
@@ -53,12 +52,13 @@ convWithWaitLog nodeId conv = conv { N.send = send', N.recv = recv' }
     MessageName sndMsg = messageName $
         ((\_ -> Proxy) :: N.ConversationActions snd rcv m -> Proxy snd) conv
 
-convWithWaitLogL
+-- TODO: Remove?
+_convWithWaitLogL
```

Comment:
Yes, if it's no longer used (and assuming we understand why it's no longer used).